docs/clustering/multi-zone-setup.rst

```diff
@@ -5,124 +5,188 @@ CrateDB multi-zone setup
 
     node.attr.zone: us-west-1
     cluster.routing.allocation.awareness.attributes: zone
 
-When the cluster is started, we will create a table with 6 shards and 1
-replica. The 6 shards will be distributed evenly across the nodes (2 shards on
-each node) and the replicas allocated on nodes with a different ``zone`` value
-to its primary shard.
+Now start your cluster and then `create a table`_ with 6 shards and 1 replica. 
+
+As an example, you can create such a table by executing a statement like this
+in the `CrateDB Shell`_:
+
+.. code-block:: sql
+
+    cr> create table my_table (
+          first_column integer,
+          second_column text
+        ) clustered into 6 shards
+        with (number_of_replicas = 1);
 
-If that isn't possible (e.g. ``num replicas > num zones - 1``) CrateDB will
+The 6 shards will be distributed evenly across the nodes (2 shards on
+each node) and the replicas will be allocated on nodes with a different
+``zone`` value than its primary shard.
+
+If this is not possible (i.e. ``num replicas > num zones - 1``), CrateDB will
 still allocate the replicas on nodes with the same ``zone`` value to avoid
-unassigned shards.
+`unassigned shards`_.
 
 .. NOTE::
 
-   Allocation awareness only means that CrateDB tries to conform to the
-   awareness attributes, to avoid such allocations, force the awareness.
+   Allocation awareness only means that CrateDB *tries* to conform to the
+   awareness attributes. To avoid such allocations, you can :ref:`force the
+   awareness <force-awareness>`.
+
+.. _force-awareness:
 
 Force awareness
 ===============
 
-To fulfill the 3rd requirement you need to ensure that when running a query on
-a node with a certain ``zone`` value it only executes the request on shards
-allocated on nodes with the same ``zone`` value.
+To fulfill the third :ref:`multi-zone requirement <multi-zone-requirements>`,
+you need to ensure that when running a query on a node with a certain ``zone``
+value, it only executes the request on `shards`_ allocated on nodes with the same
+``zone`` value.
 
 This means you need to know the different ``zone`` attribute values to force
 awareness on nodes.
 
-You can force awareness on certain attributes, for example, the ``zone``:
+You can force `awareness`_ on certain attributes with the 
+``cluster.routing.allocation.awareness.force.*.values`` setting, where ``*``
+is a placeholder for the awareness attribute, which can be defined using the
+``cluster.routing.allocation.awareness.attributes`` setting. 
+
+For example, to force awareness on the pre-configured ``zone`` attribute for
+the ``us-east-1`` and ``us-west-1`` values, you can put the following in your
+`configuration`_ file:
 
 .. code-block:: yaml
 
     cluster.routing.allocation.awareness.force.zone.values: us-east-1,us-west-1
 
-When set, no more replicas than needed are allocated on a specific group of
+This means that no more `replicas`_ than needed are allocated on a specific group of
```

Comment:
Dunno about 'on' here. Maybe 'for' or 'to'?

Reply:
I think "on" is correct here. the allocated replicas are not allocated *to* the node. they are allocated *as* replicas. the terminology is a little confusing. but this is how I understand it

@kovrus perhaps you can help clear this up? 😅